### PR TITLE
WIP: docker and docker-compose integration

### PR DESCRIPTION
**Pull request by joernroeder**
Thanks for the initial work on the Dockerfile. I was playing around with it today and got it running with the `docker-compose.yml` provided by gekko itself. This isn't "production-ready" code, more a proof of concept but works well on my local machine.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+WORKDIR /opt/japonicus/
 
 COPY . /opt/japonicus
-RUN pip3 install -r /opt/japonicus/requirements.txt
 
+EXPOSE 5000
+
+RUN python3 --version
 
-ENTRYPOINT "/bin/bash"
```
Comment (Gab0) [inline Dockerfile]: XD

**Comment by Gab0** [comment]
Nice, I hav never used docker but it seems nice.. will try to put this to work later x]

**Comment by joernroeder** [comment]
@Gab0 thanks for merging it in! as i said, it's already working but maybe you can abstract things a bit further by using dockers env variables etc.
I'll will try to deploy it to another machine i'm using as docker host and push things further as soon as i have some time to work on it again.

**Comment by joernroeder** [comment]
another feature could also to extend gekkos default `docker-compose.yml` as described here: https://docs.docker.com/compose/extends/#understand-the-extends-configuration
https://github.com/docker/compose/pull/2051

**Comment by Gab0** [comment]
#27

**Pull request merged**